fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMgr.java

```diff
@@ -38,8 +38,6 @@ public CatalogConnector createConnector(ConnectorContext context) throws StarRoc
         readLock();
         try {
             connector = ConnectorFactory.createConnector(context);
-            Preconditions.checkState(!connectors.containsKey(catalogName),
```

Comment:
The catalog/connector/metadata need to be consistent. it's better to reduce the throwing of exceptions in the creating process  :-)